locales-pending/premium.ftl

```diff
@@ -562,3 +562,11 @@ leaked-security-questions-step-two = Update them on any other site where you use
 
 # Subscription
 subscription-check-loading = Loading, please wait…
+
+# Subscriber waitlist dialog
+
+subscriber-waitlist-dialog-title = { -product-short-name } is currently at capacity
```

Comment:
```suggestion
subscriber-waitlist-dialog-title = { -brand-monitor } is currently at capacity
```

The first part of the app.ftl file is deprecated (and it would be great to have a ticket on file to clean it up…)

Reply:
Thanks! Updated the string in [c0e6825](https://github.com/mozilla/blurts-server/pull/3918/commits/c0e6825564f467012cae8488843de9eabe58ca72) and created [MNTOR-2682](https://mozilla-hub.atlassian.net/browse/MNTOR-2682).